lib/index.js

```diff
@@ -402,20 +402,28 @@ client.on('message', message => {
         // document command in help section
       case 'dotprefix':
         if (isAdmin) {
-          db.updateServerPrefix(
-            serverId, args[0]
-          ).then(data => {
-            serversConfigStore = []
-            if (data) client.guilds.cache.forEach(server => {
-              db.saveServerConfig(
-                +new Date, server.id //rework schema to eliminate the need of pushing date unecessarily here or make a find function
-              ).then(data => {
-                serversConfigStore.push(data)
+          if(args[0] != undefined) {
+            if(args[0].length == 1) {
```

Comment:
- Instead of two if statements one is sufficient.
- Optional chaining can be used instead of checking whether value is defined.
- Type check should be used for equality/unequality statements.

```suggestion
            if (args[0]?.length === 1) {
```